pkg/chartverifier/checks/helm.go

```diff
@@ -175,3 +186,45 @@ func IsChartNotFound(err error) bool {
 	_, ok := err.(ChartNotFoundErr)
 	return ok
 }
+
+func getImages(chartUri string) ([]string, error) {
```

Comment:
the function returns image references, so let's name it so - e.g. `getImageReferences`

Reply:
Renamed.